src/index.js

```diff
@@ -1137,6 +1139,23 @@ module.exports = (function() {
 
                         // Add the cell that previously held the placeholder
                         newRow.append(newCell);
+
+                        if(mergeCell) {
+                            var mergeRange    = self.splitRange(mergeCell.attrib.ref),
```

Comment:
can you use `let` instead of `var` and make each variable as separate variable on individual line.

Reply:
Ok! I'll work on it

Reply:
Do you have a chance to work on the suggestions?

Reply:
Hello, 
Thanks for very useful library.
I really need this feature.
Can I know why not merged yet?

Reply:
@muyoungko can I assume that you take care of this scenario in #188 ?

Reply:
Yes~